python/GangaLHCb/Lib/GaudiRunApp/GaudiRunApp.py

```diff
@@ -0,0 +1,250 @@
+
+from Ganga.GPIDev.Adapters.IPrepareApp import IPrepareApp
+from Ganga.GPIDev.Schema import Schema, Version, SimpleItem, GangaFileItem
+
+from Ganga.GPIDev.Lib.File.File import ShareDir
+from Ganga.Core import ApplicationConfigurationError, GangaException
+
+from Ganga.Utility.logging import getLogger
+
+from Ganga.GPIDev.Base.Proxy import getName
+
+from os import rename, path, makedirs, chdir
```

Comment:
Yes but there is an extremely _large_ need to do this in an automated way across a lot (everywhere else) in the codebase.
Doing this in 2 files will help the PR but I don't want to have to do this every time I open a file to fix a typo whilst I'm doing other work and I'm unaware of any large PR to fix this.

There is a lack of compliance because we haven't set a standard. Can we please add something to the dev docs to describe how we should work with the code. Preferably with some command to highlight all of these things in a useful way for the dev so that these things can be worked out without a lot of to and fro and a lot of emails? I'm happy to fix things but a back and forth on github is less than ideal in the long-term I think.
Changes incoming.